### DISCUSSION TITLE
Getting server not found error on dockerized qfieldcloud instance.

### DISCUSSION THREAD

**Discussion by amitabhajana** [opengisch/QField#3754] [Q&A]
I have deployed a qfieldcloud instance on my machine using docker-compose. But when i am trying to access the application on web browser, I'm getting server not found error, But i can open the django admin and swagger portal perfectly. Error screenshots are attatched.
![Screenshot from 2022-12-19 17-13-39](https://user-images.githubusercontent.com/70099241/208420081-1aca3cfe-b8b2-41a6-8823-11266a0bf646.jpg)

**Comment by suricactus** [comment]
Please be more precise about the steps followed. Hiding the URI part of the URL didn't help. I guess you are looking for https://hiddendomainfornoreason.com/admin .

**Reply by amitabhajana** [reply]
I have deployed a qfieldcloud instance in production environment on my local machine.
What i'm trying to say is like when i am openning the application in browser by using the url http://localhost/ i am getting the above error.
But i can access the django admin portal by using the url http://localhost/admin .

**Reply by suricactus** [reply]
I think you have misleading title, it's not "server not found", but "page not found", or 404 error page.

And it is actually very much correct status code, this repo provides very few paths:
- /admin/*
- /api/*
- and the rest is here https://github.com/opengisch/qfieldcloud/blob/master/docker-app/qfieldcloud/urls.py

If you are interested in on premise installation, please drop us an email info [at] opengis.ch
